.github/workflows/stale.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */4 * * *'
+
+jobs:
+  prune_stale:
+    name: Prune Stale
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Prune Stale
+      uses: actions/stale@v3.0.14
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        # Different amounts of days for issues/PRs are not currently supported but there is a PR
+        # open for it: https://github.com/actions/stale/issues/214
+        days-before-stale: 30
+        days-before-close: 7
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it has not had activity in the
+          last 30 days. It will be closed in the next 7 days unless it is tagged "help wanted" or "no stalebot" or other activity
```

Comment:
Suggestion: Several projects used the `pinned` label for this purpose

@artursouza Whatever the name, a maintainer needs to add the appropriate label for the documentation to be accurate.

Reply:
It maybe need to discuss which label we use.

Reply:
Let's use the `pinned` label.

Reply:
When this runs for the first time, will it close all issues and PRs right away or will it give the 7 days notice first? I want to know if we want to label the existing PRs and issues now or later.

Reply:
I am also adding the `triage/resolved` label as exception.

Reply:
@artursouza should add label now.

Reply:
Sorry, I just realized there are two workflows. Do we still need stale.yml? I thought we were merging all in one.

Reply:
Sure, I will merge them.